qiling/const.py

```diff
@@ -36,11 +36,12 @@ class QL_OS(IntEnum):
     BLOB = 210
 
 class QL_VERBOSE(IntEnum):
-    OFF = 0
-    DEFAULT = 1
-    DEBUG = 4
-    DISASM = 10
-    DUMP = 20
+    OFF = -1    # turn off all the output
+    MINIMUM = 0 # output only warnings 
```

Comment:
This is causing a break change. Keep the value of `OFF` and assign another value for `-1` instead.